code/drasil-example/drasil-example.cabal

```diff
@@ -96,7 +96,7 @@ executable nopcm
     drasil-code >= 0.1.1,
     drasil-printers >= 0.1.0,
     drasil-gen >= 0.1.0,
-    drasil-docLang >= 0.1.1
+    drasil-docLang >= 0.1.2
```

Comment:
I bumped NoPCM's docLang requirement since it now uses mkEnumCC, but I'm not sure protocol for updating the version number for drasil-examples itself. Should I have bumped it in this commit?

Reply:
If `drasil-examples` uses `mkEnumCC`, then yes. Until it does, then not necessary, and the bump can occur in that commit.